src/components/Header.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const Header = () => (
+  <header className="flex-row">
+    <h1>Math Magicians</h1>
+    <nav>
+      <ul className="flex-row">
+        <li>
+          <a href="/">Home</a>
+        </li>
+        <li>
+          <a href="/calculator">Calculator</a>
+        </li>
+        <li>
+          <a href="/quote">Quote</a>
+        </li>
```

Comment:
- [ ] Same here, Please use react router `<Link>` tag instead of `<a>` tag to make your app act like SPA.
It will be like that:
```javascript
        <li>
          <Link to="/">Home</Link>
        </li>
        <li>
          <Link to="/calculator">Calculator</Link>
        </li>
        <li>
          <Link to="/quote">Quote</Link>
        </li>
```
See this [article](https://www.freecodecamp.org/news/how-to-use-react-router-version-6) for more info.

Reply:
@Omar-Muhamad Thank you very much